# Message for requesting path and passing obstacle info as point cloud
Header header
geometry_msgs/PoseWithCovariance pose
sensor_msgs/PointCloud pointcloud
